misc-scripts/xref_mapping/XrefParser/WormbaseCElegansBase.pm

```diff
@@ -1,6 +1,6 @@
 =head1 LICENSE
 
-Copyright [2018-2021] EMBL-European Bioinformatics Institute
+Copyright [2018-2020] EMBL-European Bioinformatics Institute
```

Comment:
Hi Mag. Is there a reason the license here was reverted to 2020? (WormbaseCElegansRefSeqGPFFParser.pm and WormbaseCElegansUniProtParser.pm)

Reply:
Well spotted, those files were wrongly removed and I seem to have found an older copy where the copyright was out of data
sorted now